pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -143,7 +143,7 @@ stdenv.mkDerivation rec {
 
   postFixup = ''
     # the .desktop is both invalid and pointless
-    rm $out/share/applications/qemu.desktop
+    rm $out/share/applications/qemu.desktop || true
```

Comment:
rf -f ?

Reply:
``rm -f`` means we won't notice when it is removed. Same with ``|| true``.

Reply:
We will have to either use `|| true`, `-f` or make the nix code emit different code here. That is that I understand the reasoning right. Some of our builds are not producing `.desktop` files and thus we must choose one of those options. The suggestion from @SuperSandro2000 would make this diff a no-op.

Reply:
> Some of our builds are not producing .desktop files

Did not know that. Maybe add a comment about this.